rise-google-sheet.html

```diff
@@ -212,7 +212,7 @@
       },
 
       _getDataKey: function() {
-        return DATAKEY_BASE_NAME + "_" + this.key + "_" + this.sheet +
+        return DATAKEY_BASE_NAME + "_" + this.key + "_" + encodeURIComponent( this.sheet ) +
```

Comment:
@stulees I saw it could not save to RC. It will save to localstorage though. So it will not break it will create a new entry point on localstorage and finally add the data to RC. cheers